base/docs/helpdb/Base.jl

```diff
@@ -549,26 +549,24 @@ to synchronous `File`'s and `IOStream`'s not to any of the asynchronous streams.
 """
 fd
 
+
 """
-    ones(type, dims)
+    ones([A::AbstractArray,] [T=eltype(A)::Type,] [dims=size(A)::Tuple])
 
-Create an array of all ones of specified type. The type defaults to `Float64` if not specified.
+Create an array of all ones with the same layout as `A`, element type `T` and size `dims`.
+The `A` argument can be skipped, which behaves like `Array{Float64,0}()` was passed.
+For convenience `dims` may also be passed in variadic form.
```

Comment:
There are examples where `dims...` is used already e.g. `zeros(Int8, 2, 3)`. But I can open a new PR with stevengj's suggestion if you like.